.circleci/config.yml

```diff
@@ -330,6 +330,7 @@ jobs:
                 --debug --write-graph --use-syn-sdc --use-aroma \
                 --ignore-aroma-denoising-errors --mem_mb 4096 \
                 --output-space T1w template fsaverage5 \
+                --template-resampling-grid 2mm \
```

Comment:
How about we set this option on to `native` on ds054, just so that if/when we change this, we don't have to remember to set one back to native for testing.

Reply:
done